ResearchKit/Common/ORKResult.m

```diff
@@ -178,6 +178,99 @@ - (NSString *)description {
 @end
 
 
+@implementation ORKToneAudiometryResult
+
+- (void)encodeWithCoder:(NSCoder *)aCoder {
```

Comment:
Ah. This is missing JSON serialization support for both result and step in ORKESerialization. In any case, you need to run the tests attached to the ORKTest testing app.